src/mlte/_private/job.py

```diff
@@ -0,0 +1,56 @@
+import sys
```

Comment:
Can we add a brief file-level comment that describes the content of this file?

Reply:
Sure! We can restructure or move this somewhere else if needed, I guessed this could be a good location.